content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2023
+linkTitle: KubeCon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
+  Sharma Dyla Benedikt Bongartz Oliveira Pathak
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee
+and technical committee are excited to be at [KubeCon EU][] in Amsterdam from
+April 18 - 21, 2023!
+
+You can meet us at the [project booth][], listen to our talks or learn with us
```

Comment:
As an end-user reading this, I immediately wonder: when and where can I meet folks at the booth? Is that every day all day? Where?

Reply:
I changed the intro to _Read on to learn about all the things related OpenTelemetry during KubeCon._ and leave it to the rest of the content to guide people to the right dates and locations, wdyt?